packages/ssr/scripts/utils.ts

```diff
@@ -90,10 +94,42 @@ const ${toPascalCase(elementName)}WebComponent = React.forwardRef(({ children =
 
 function ${toPascalCase(elementName)}(props) {
   if (!props['noSSR'] && typeof window == 'undefined') {
+    let cloned = [];
+
+    // Pass wrapped prop to children GCDS components
+    if (Array.isArray(props.children)) {
+      for (let x = 0; x< props.children.length; x++) {
```

Comment:
You could simplify the structure a little bit if you use a map function for the array iteration instead.